docs/protocol-design/work.md

```diff
@@ -1,6 +1,6 @@
-Title: Nano Protocol Design - Proof of Work
+Title: Nano Protocol Design - Proof-of-Work
```

Comment:
Due to the confusion that calling this "Proof-of-Work" brings with regards to other coins, mining, etc. we have discussed possibly shifting from using this term. Some suggestions have been made, I am particularly in favor of just calling it "work" as that is what is already included in RPC commands and captures the nature of it. Although we would still want to call out that the methods used to generate work are similar to Proof-of-Work coins, perhaps this is a good time to shift that language to "work" or another term.